deploy/osd-project-request-template/02-role.dedicated-admins-project-request.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: dedicated-admins-project-request
```

Comment:
Can't take this out, it removes management of the template, which we're trying to allow.